content/vgg-classifier-tutorial.md

```diff
@@ -0,0 +1,320 @@
+Title: Keras, Make your own image classifier
+Date: 2016-03-30
+Category: Tutorial
+Author: Gregory Senay
+
+
+This tutorial explains how to create image classifier, from an ImageNet pre-trained model, your own model on your own dataset.
+
+Prepare your dataset
+-----
+Here, I will explain how to simply prepare your image dataset following an existing simple one: Caltech101.
+Caltech101 is a dataset of 101 categories (crocodile, camera, plane, soccer ball...) with 40 to 800 images per category, the size of image is bout 200x300.
+To prepare your data, you have to follow this directory architecture:
+```
+  * Main Directory # Name of you dataset
+    * Class1Name
+      * Image1Class1
```

Comment:
It would be nice to specify that `Class1Name` is a directory (e.g. by writing it as `Class1Name/`) and specify that `Image1Class1` are picture files (e.g. by writing it as `Image1Class1.jpg`)